libraries/lib-time-and-pitch/StaffPad/TimeAndPitch.cpp

```diff
@@ -44,12 +44,15 @@ inline float lagrange6(const float (&smp)[6], float t)
   return (c0 + c1 * t) + (c2 + c3 * t) * t2 + (c4 + c5 * t) * t2 * t2;
 }
 
-std::vector<float> generateRandomPhaseVector(size_t size)
+std::vector<std::complex<float>> generateRandomPhaseVector(size_t size)
```

Comment:
Paul has introduced the class for aligned vectors, hasn't he?

Reply:
Not necessary here, due to [this](https://github.com/audacity/audacity/pull/6068#discussion_r1521902915).